app/templates/components/component-tree-item.hbs

```diff
@@ -21,14 +20,46 @@
   {{/if}}
 
   <code
-    class="component-tree-item__tag {{if @item.isComponent "component-tree-item__bracket"}} {{if (not @item.hasChildren) "component-tree-item__bracket--self-closing"}}"
+    class="component-tree-item__tag
+      {{if
+        (and (not @item.args.positional) @item.isComponent)
+        "component-tree-item__bracket"
+      }}
+      {{if
+        (and (not @item.args.positional) (not @item.hasChildren))
```

Comment:
Still need `@item.isComponent`. Alternatively, we can make this `component-tree-item__no-children` and deal with the composition in CSS.